daniddelrio/akads-ph
# Add emailing feature

To test:
- MAKE SURE YOU HAVE THE .ENV FILE
1. Make sure you have a tutor account and a tutee account WITH AN EMAIL YOU HAVE ACCESS TO.
2. Book a session using a tutee account.
3. Accept the session using a tutor account.
4. Using Tutor: Click Sessions in the Navbar > Upcoming. Click Complete Session.
Input a date, start time, and end time indicating the details of the session. Click Complete Session after.
5. Using Tutee: Click Sessions in the Navbar > Unconfirmed. The session should appear there. Click Confirm now to see if the payment amount is finalized.
6. After clicking confirm, check the email of the tutee account to see if an email was sent and the details are correct.
